frontend/src/views/PlayToEarn.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div class="body main-font">
+    <div>
+      <div class="row">
+          <div class="col-lg-6 col-sm-12 adventure">
+            <h5>PLAY TO EARN</h5>
+          </div>
+      </div>
+      <img src="../assets/divider7.png" class="info-divider enemy-divider" />
+      <div class="row">
+          <div class="col-lg-12 play-to-earn">
+            <router-link v-if="!stakeOnly" :to="{ name: 'combat' }" exact class="adventures navs">
+                <div>
+                  <h4>ADVENTURE</h4>
+                  <p>Engage on  PVE battles to earn rewards.</p>
+                  <div class="btn-join">COMBAT</div>
+                </div>
+            </router-link>
+
+            <!-- change this to raid route -->
+            <router-link v-if="!stakeOnly" :to="{ name: 'combat' }" exact class="raid navs">
+                <div>
+                  <h4>RAID</h4>
+                  <p>Join the players in defeating the boss.</p>
+                  <div class="btn-join">JOIN</div>
+                </div>
+            </router-link>
+
+            <!-- change this to arena route -->
+            <router-link v-if="!stakeOnly" :to="{ name: 'combat' }" exact class="arena navs">
+                <div>
+                  <h4>ARENA</h4>
+                  <p>Defeat others Players on PVP battles</p>
```

Comment:
Please add all these to the translation file

Reply:
> Please add all these to the translation file

ok will do

Reply:
done